.github/native-tests.json

```diff
@@ -62,8 +62,8 @@
         },
         {
             "category": "Messaging2",
-            "timeout": 70,
-            "test-modules": "artemis-core, artemis-jms, reactive-messaging-amqp, reactive-messaging-http, reactive-messaging-rabbitmq",
+            "timeout": 110,
```

Comment:
Ok, that's a BIG bump. 75 or 80 should have been fine.

Reply:
I just matched the `Messaging1` category above.